code/workspaces/client-api/src/dataaccess/usersService.js

```diff
@@ -39,10 +39,24 @@ async function isMemberOfProject(projectKey, token) {
   return exists;
 }
 
+async function setInstanceAdmin(userId, instanceAdmin, token) {
+  logger.debug(`Setting instanceAdmin of ${userId} to ${instanceAdmin}`);
+  const body = { instanceAdmin };
+  const { data } = await axios.put(`${authServiceUrl}/roles/${userId}/instanceAdmin`, body, generateOptions(token));
+  return { userId, instanceAdmin: data.instanceAdmin };
```

Comment:
It would then clash with the argument, which obviously I could change to not be instanceAdmin, but I thought referring to the returned instanceAdmin as data.instanceAdmin was helpful in keeping the distinction.

Reply:
Ahhh I completely missed that!